File: 1791-richest-customer-wealth/1791-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        PriorityQueue<Integer> pq = new PriorityQueue<>((a,b)->b-a);
        List<Integer> l = new ArrayList<>();
        for(int i=0;i<accounts.length;i++){
            int sum = 0;
            for(int j=0;j<accounts[i].length;j++){
                sum +=accounts[i][j];
            }
            pq.offer(sum);
        }
        return pq.poll();
    }
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        ans=[]
        length = len(accounts)
        for i in range(length):
            innerlength = len(accounts[i])
            sum = 0
            for j in range(innerlength):
                sum +=accounts[i][j]
            ans.append(sum)
        ans.sort()
        return ans[-1]